instructor/ui/instructor.js

```diff
@@ -163,6 +163,30 @@ $(function(){
     $("#id_has_nuclear_fractination").attr('disabled', true).parent().addClass('scb_ab_s_grayed');
     $("#id_has_cytoplasmic_fractination").attr('disabled', true).parent().addClass('scb_ab_s_grayed');
 
+    /**
+     * Western Blotting: band intensity
+     */
+    var sliders = $(".intensity_slider");
+    for(var i=0; i<sliders.length;i++){
+        var $slider=$(sliders[i]);
+        var intensity_dict={0:'', 0.5:'low', 1:'med', 1.5:'high', 2:'extra high'};
+        var intensity=$($slider.siblings('input')[0]).attr('value');
+        $($slider.parent().siblings('.scb_ab_s_intensity_text')).html(intensity_dict[intensity]);
```

Comment:
It's safer to use `.text(intensity_dict...` instead of `.html` if you aren't intending to put any HTML in the element.